suites/pacific/rados/tier-3_rados_ssd-tests.yaml

```diff
@@ -65,12 +65,43 @@ tests:
                   service_id: ssd_nodes
                   placement:
                     nodes:
-                      - node1
-                      - node2
-                      - node3
+                      - node4
+                      - node5
+                      - node6
+                      - node7
+                      - node8
+                      - node9
+                      - node12
+                      - node13
                   spec:
```

Comment:
Here we can use limit flag to deploy OSDs on limited number of SSD drives so that some as left as spare drives for other tests.
```
spec:
  data_devices:
    rotational: 0
    limit: <>
```

Reply:
There is only one SSD available on the Mero nodes. node14 and node15 SSDs are used for other tests.

[root@mero001 ubuntu]# lsblk -d -o name,rota
NAME    ROTA
sda        1
sdb        1
sdc        1
sdd        1
sde        1
sdf        1
sdg        1
sdh        1
sdi        1
sdj        1
sdk        1
sdl        1
sdm        0
nvme0n1    0
[root@mero001 ubuntu]#